docker-compose-common.yml

```diff
@@ -39,9 +39,9 @@ loader:
 
 # Must be named "elasticsearch" to create expected ELASTICSEARCH_* envvars
 elasticsearch:
-  image: elasticsearch
+  image: elasticsearch:1.4.5
 
```

Comment:
We're about to upgrade to version 1.5. Not an issue at this point I think, but might be worth considering upgrading